docs/data/base/components/button/button.md

```diff
@@ -1,17 +1,49 @@
 ---
 product: base
-title: React button unstyled
+title: Unstyled React button
+components: ButtonUnstyled
 githubLabel: 'component: button'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#button
 ---
 
-# Button unstyled
+# Unstyled button
 
 <p class="description">Buttons allow users to take actions, and make choices, with a single tap.</p>
```

Comment:
```suggestion
<p class="description">Buttons allow users to take actions and make choices with a single tap.</p>
```

@samuelsycamore I'm not sure about the second comma, though. Could you please verify?